app/webpacker/components/RegistrationsV2/Register/RegistrationClosedMessage.jsx

```diff
@@ -0,0 +1,48 @@
+import { DateTime } from 'luxon';
+import React, { useEffect, useState } from 'react';
+import { Message } from 'semantic-ui-react';
+import I18nHTMLTranslate from '../../I18nHTMLTranslate';
+
+export default function RegistrationClosedMessage({
+  registrationStart,
+  onTimerEnd,
+}) {
+  const [timeLeft, setTimeLeft] = useState({
+    days: 0, hours: 0, minutes: 0, seconds: 0,
+  });
+
+  const start = DateTime.fromISO(registrationStart);
+
+  // Update the timer every second
+  useEffect(() => {
+    const updateTimer = () => {
+      const now = DateTime.local();
+
+      if (now >= start) {
+        onTimerEnd();
+        return;
+      }
```

Comment:
Not a big deal, but it's best to have each effect do 1 effect, so I would separate this onto its own effect (dependent on all the time values - if they're all 0 then call `onTimerEnd`). (Which is an example of why they should be initialized properly.)

Reply:
I added the second effect